src/SwaggerProvider.DesignTime/DefinitionCompiler.fs

```diff
@@ -8,6 +8,10 @@ open Microsoft.FSharp.Quotations
 open System
 open System.Reflection
 
+type FileWrapper(fileName: string, data: IO.Stream) =
```

Comment:
I tried to make this a tuple of these two pieces of data but that failed for some reason.